prometheus/prometheus.go

```diff
@@ -109,7 +113,17 @@ func ResolverMiddleware() graphql.FieldMiddleware {
 			exitStatus = exitStatusSuccess
 		}
 
-		timeToResolveField.WithLabelValues(exitStatus, rctx.Object, rctx.Field.Name).
+		var errCode string
+		if err != nil {
+			gqlErr := &gqlerror.Error{}
```

Comment:
nit: `var gqlErr *gqlerror.Error`

Reply:
>  second argument to errors.As must be a non-nil pointer to either a type that implements error, or to any interface type
code from gqlerror pkg, error.go:
```go
func (err *Error) Error() string {...}
```
only ptr on `Error` implement std error

Reply:
``` 
```var gqlErr *gqlerror.Error
errors.As(err, &gqlErr)

is a non nil pointer. It's a pointer of **gqlerror.Error type to nil value of *gqlerror.Error

Reply:
the main idea of this: you don't need to allocate a whole type value if error type is different

Reply:
yep, u are right!

Reply:
fixed